rest-service/manager_rest/rest/resources_v3_1/blueprints.py

```diff
@@ -91,10 +91,12 @@ def put(self, blueprint_id, **kwargs):
         """
         rest_utils.validate_inputs({'blueprint_id': blueprint_id})
 
-        args = (
-            json.loads(request.form.get('params', '{}'))
-            or dict(request.args)
-        )
+        args = None
+        form_params = request.form.get('params')
+        if form_params:
+            args = json.loads(form_params)
+        if not args:
```

Comment:
This should maybe be 'if args is not None', given that args could be {} if form params were passed in, but empty?
Are we likely to care about that condition?

Reply:
I note that it was the same before this change! But also, well, why not. Might as well check args.